src/processor/prometheustypeconverterprocessor/go.mod

```diff
@@ -1,46 +1,46 @@
 module github.com/solarwinds/swi-k8s-opentelemetry-collector/processor/prometheustypeconverterprocessor
 
-go 1.17
+go 1.18
```

Comment:
1.18 or 1.19 ?

Reply:
The latest otel collector packages are using 1.18.